recipes/r-ggthemes/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = '3.4.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-ggthemes
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: ggthemes_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/ggthemes_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/ggthemes/ggthemes_{{ version }}.tar.gz
+  sha256: 1a1aeb7fe5a60fab0d0b82e959dd7c01d80b6f5984af77674264cc25cfa548ca
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: extrafont, knitr, lintr, maps, mapproj, pander, plyr, reshape2, rmarkdown, testthat, tidyverse
+requirements:
+  build:
+    - r-base
+    - r-assertthat
+    - r-colorspace
+    - r-ggplot2 >=2.2.0
+    - r-scales
+
+  run:
+    - r-base
+    - r-assertthat
+    - r-colorspace
+    - r-ggplot2 >=2.2.0
+    - r-scales
+
+test:
+  commands:
+    - $R -e "library('ggthemes')"  # [not win]
+    - "\"%R%\" -e \"library('ggthemes')\""  # [win]
+
+about:
+  home: http://github.com/jrnold/ggthemes
+  license: GPL-2
```

Comment:
GPL requires that the license be distributed with the package. Please include a `license_file` entry.